internal/server/data/grant.go

```diff
@@ -176,11 +176,11 @@ func ListGrants(tx ReadTxn, opts ListGrantsOptions) ([]models.Grant, error) {
 			for _, id := range groupIDs {
 				subjects = append(subjects, uid.NewGroupPolymorphicID(id).String())
 			}
-			query.B(`AND subject IN (?)`, subjects)
+			query.B(`AND subject = any(?)`, subjects)
```

Comment:
maybe query.B should just look for `IN (?)` paired with a slice and auto-convert it to `IN (?, ?, ?)` etc. Feels very weird to force the dev to use different sql. At the very least we can auto-convert it to the `any` syntax if that is identical.

Reply:
Ok, I've implemented `queryInClause` which looks like the second example above, and removed all of the `= any(?)` and `<> all(?)`.